app/src/main/java/com/doubtless/doubtless/screens/answers/viewholder/AnswerViewHolder.kt

```diff
@@ -64,6 +64,13 @@ class AnswerViewHolder(itemView: View, private val interactionListener: Interact
         description.text = answerData.description
         tvYear.text = "| ${answerData.authorYear} Year |"
 
+        if (answerData.authorId == DoubtlessApp.getInstance().getAppCompRoot().getUserManager()
+                .getLoggedInUser()?.id
```

Comment:
you should take user from cached user, this method hits the storage everytime its called.

Reply:
fixed